optimum/intel/openvino/modeling_base.py

```diff
@@ -264,6 +264,10 @@ def _from_transformers(
             "trust_remote_code": trust_remote_code,
         }
 
+        if model_id.startswith("timm/"):
+            model_kwargs["use_safetensors"] = False
+            model_kwargs["num_labels"] = config.num_classes
+            
```

Comment:
Also could you add a test to verify that `timm` models are indeed compatible with `OVModels` ? I wonder if we shouldn't keep the `OVModelForXxx` API for `transformers` models and have our own `timm.create_model` for `timm` models

Reply:
@AlexKoff88 @helena-intel what do you think ?

Reply:
I'm not familiar with `timm` yet. I love that optimum-intel allows using transformer models with only minimum changes to code (change AutoModelFor... to OVAutoModelFor...) and it would be great to have that for `timm` too, that existing `timm` users can just change a tiny bit of code and leverage OpenVINO.  Just changing the timm.create() line in the model card from https://huggingface.co/timm/vit_tiny_patch16_224.augreg_in21k for example.

Though if it is possible to load classification models with OVModelForImageClassification, that would be so much easier than "manual" inference. But it seems you would need support for that, with a configured preprocessor for the model. It doesn't seem like this is supported/expected for these models at the moment though, and it seems it can lead to issues to use it this way.

Reply:
> Also could you add a test to verify that `timm` models are indeed compatible with `OVModels` ? I wonder if we shouldn't keep the `OVModelForXxx` API for `transformers` models and have our own `timm.create_model` for `timm` models

This is definitely down the pipeline, as I plan to also add/test some more model optimization on timm models, during my project. Do you suggest test for any specific feature for OVModels? Because, you can see in the accompanying notebook, that after loading through OVModelForImageClassification, the model gives output of correct size.

Reply:
thanks @sawradip ! what do you think about adding tests in [`test_modeling.py`](https://github.com/huggingface/optimum-intel/blob/v1.10.1/tests/openvino/test_modeling.py) to compare the original timm models with one which was exported to OpenVINO ?

Reply:
This is a great idea @echarlaix . I will look into creating test cases. For now I am working for an alternate architecture to support more timm models in [this](https://github.com/sawradip/optimum-intel/tree/timm_integration_custom) branch . As soon I can prepare the PR, I will add the tests and let you know.